src/main/java/uk/gov/companieshouse/confirmationstatementapi/model/ActiveOfficerDetails.java

```diff
@@ -21,6 +21,18 @@
     private Address serviceAddress;
     @JsonProperty("residential_address")
     private Address residentialAddress;
+    private boolean corporate;
```

Comment:
does the json output put an 'is' on the corporate field name? if not, see if you can use `@JsonProperty(is_corporate)` here